src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,56 @@
+package core.basesyntax;
+
+import java.text.DecimalFormat;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int firstParallel;
+    private int secondParallel;
+    private int height;
+
+    public IsoscelesTrapezoid(String color, int firstParallel, int secondParallel, int height) {
+        super(color);
+        this.firstParallel = firstParallel;
+        this.secondParallel = secondParallel;
+        this.height = height;
+    }
+
+    public int getFirstParallel() {
+        return firstParallel;
+    }
+
+    public void setFirstParallel(int firstParallel) {
+        this.firstParallel = firstParallel;
+    }
+
+    public int getSecondParallel() {
+        return secondParallel;
+    }
+
+    public void setSecondParallel(int secondParallel) {
+        this.secondParallel = secondParallel;
+    }
+
+    public int getHeight() {
+        return height;
+    }
+
+    public void setHeight(int height) {
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return ((firstParallel + secondParallel) / 2) * height;
```

Comment:
`(firstParallel + secondParallel) / 2`: integer division in floating-point context .
Please pay attention to [this](https://www.javatpoint.com/java-int-to-double) article.